### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by bottrall**
This PR adds a `SECURITY.md` to our community health files.

### CODE REVIEW

**Comment by bottrall** [comment]
WIP. Adapted from [GitHub's SECURITY.md](https://github.com/github/.github/blob/main/SECURITY.md).

**Comment by bobop** [comment]
Thanks for this @jake-bottrall 👍 

I'd suggest connecting with the PANDAS about the email addresses to promote - we have security@jane.app and privacy@jane.app which exist for different purposes. Also, they will have the Policy content and may have some other copy to add in an email template for replying to security report emails.

**Comment by bottrall** [comment]
Thanks, @bobop, reached out to [PANDAS](https://janeapp.slack.com/archives/CDUECEMLL/p1707238222105679).

**Comment by phil-janeapp** [comment]
Looks like a reasonable start to me :+1: thanks for doing this @jake-bottrall

**Comment by bottrall** [comment]
[Discussed with Panda's](https://janeapp.slack.com/archives/CDUECEMLL/p1707238222105679), they're happy for security@jane.app to be the provided email address & as of right now Jane does not have a policy around reporting security vulnerabilities. So I'm omitting it all together until there's a need for it.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Security
+
+Jane takes the security of our software products and services seriously, including all of the open source code repositories managed through the [Jane](https://github.com/janeapp) organization.
+
+## Reporting Security Issues
+
+If you believe you have found a security vulnerability in any Jane-owned repository, please report it to us through coordinated disclosure.
+
+**Please do not report security vulnerabilities through public Jane issues, discussions, or pull requests.**
```
Comment (phil-janeapp) [inline SECURITY.md:11]: This should be self-explanatory, but do we want to explicitly call out spaces such as "social media" or "the Jane Facebook community" here?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Please include as much of the information listed below as you can to help us better understand and resolve the issue:
+
+  * The type of issue (e.g., buffer overflow, SQL injection, or cross-site scripting)
+  * Full paths of source file(s) related to the manifestation of the issue
+  * The location of the affected source code (tag/branch/commit or direct URL)
+  * Any special configuration required to reproduce the issue
+  * Step-by-step instructions to reproduce the issue
+  * Proof-of-concept or exploit code (if possible)
+  * Impact of the issue, including how an attacker might exploit the issue
```
Comment (phil-janeapp) [inline SECURITY.md:23]: Possibly add:

- If possible, a **privately-shared** Github repository or script containing the full steps to replicate the issue would be incredibly helpful

**Review by phil-janeapp — APPROVED** [review]
A few potential additions, but looks good as-is.

File: SECURITY.md (hunk 1 above)
Comment (bottrall) [reply SECURITY.md:11]: It's a good point. But I'm also wondering if it's necessary due to the context of anyone reading is on Jane's github organization 🤔 I could go either way on it.

File: SECURITY.md (hunk 2 above)
Comment (bottrall) [reply SECURITY.md:23]: 2073d3806f5ce638f61bcd6311a5abc5a514018c

**Review by hollandmatt — APPROVED** [review]
Sounds good! 🔐

**Review by andymckay — APPROVED** [review]

**Review by whyakshay — APPROVED** [review]

**Pull request merged**